packages/atlas-react/src/components/AtlasAlert/AtlasAlert.scss

```diff
@@ -0,0 +1,81 @@
+@import "~@productplan/atlas-styles/atlas.scss";
+
+.AtlasAlert{
+  border-radius: 4px;
+  display: flex;
+  flex-direction: row;
+  width: 548px;
+  
+  &__body {
+    margin:  16px 16px 16px 8px;
+    font-size: 14px;
+    line-height: 20px;
```

Comment:
literally the only thing in this PR I can give ANY feedback on is the font-size + line height.

I would pick `--a-line-height-font-body-20` and `--a-size-font-body-14` in this case. I agree these could definitely have better names, and the use of rem units is curious.

Reply:
current:
![Screen Shot 2023-02-28 at 11 43 48 AM](https://user-images.githubusercontent.com/56092831/221961921-c44edfc3-1156-43ec-8d61-cf8b0a2ebb69.png)

with changes: 
![Screen Shot 2023-02-28 at 11 42 06 AM](https://user-images.githubusercontent.com/56092831/221961954-79ebefc7-0b3c-4b5d-83a3-51c3847eb50c.png)

I think the base size is 16px so the line height spacing is off with changes.  
``` 
     --a-size-font-body-14: 0.875rem -> 0.875(16px) = 14px
     --a-line-height-font-body-20: 1.875rem -> 1.875(16px) = 30px 
```